docs/topics/js/js-project-setup.md

```diff
@@ -1,4 +1,4 @@
-[//]: # (title: Set up a Kotlin/JS project)
+[//]: # (title: Understand a Kotlin/JS project)
```

Comment:
This page isn't about Understanding the project.
It shows how to set up JS-related Gradle props. I'd like you to rollback the changes (or start a discussion on why it should be named "Understand").

Reply:
We can keep `Set up`.

Reply:
Done ✅